openvdb_houdini/SOP_OpenVDB_Merge.cc

```diff
@@ -0,0 +1,296 @@
+        SOP_OpenVDB_Merge::theSOPTypeName,
+        "VDB Merge",
+        SOP_OpenVDB_Merge::myConstructor,
+        SOP_OpenVDB_Merge::buildTemplates(),
+        1,
+        OP_MULTI_INPUT_MAX,
+        nullptr,
+        0));
+}
+
+
+static const char *theDsFile = R"THEDSFILE(
+{
+    name        parameters
+
+    inputlabel  1   "VDBs To Merge"
+
+    parm {
+        name    "group"
+        label   "Group"
+        type    string
+        default { "" }
+    }
+    parm {
+        name    "vdbpointsgroup"
+        label   "VDB Points Group"
+        type    string
+        default { "" }
+    }
+    parm {
+        name    "collation"
+        label   "Collation"
+        type    ordinal
+        default { "nameclassandtype" }
+        menu {
+            "nameclassandtype"  "Grid Name, Grid Class and Value Type"
+            "nameandclass"      "Grid Name and Grid Class"
+            "classandtype"      "Grid Class and Value Type"
+        }
+    }
+    parm {
+        name    "usereferencevdb"
+        label   "Use First Grid as Reference VDB Only"
+        type    toggle
+        default { "0" }
+    }
+    parm {
+        name    "resample"
+        label   "Resample Grids"
+        type    ordinal
+        default { "first" }
+        menu {
+            "first"         "To Match First Grid"
+            "highestres"    "To Match Highest-Res Grid"
+            "lowestres"     "To Match Lowest-Res Grid"
+        }
+        joinnext
+    }
+    parm {
+        name    "resampleinterp"
+        label   "Interpolation"
+        type    ordinal
+        default { "linear" }
+        menu {
+            "point"     "Nearest"
+            "linear"    "Linear"
+            "quadratic" "Quadratic"
+        }
+    }
+    groupsimple {
+        name    "mergemodegroup"
+        label   "Merge Mode"
+
+        parm {
+            name    "fogmode"
+            label   "Fog Mode"
+            type    ordinal
+            default { "add" }
+            menu {
+                "none"          "None"
+                "add"           "Add"
+                "multiply"      "Multiply"
+                "maximum"       "Maximum"
+                "minimum"       "Minimum"
+                "topounion"     "Activity Union"
+                "topointersect" "Activity Intersection"
+            }
+            joinnext
+        }
+        parm {
+            name    "scalarmode"
+            label   "Scalar Mode"
+            type    ordinal
+            default { "add" }
+            menu {
+                "none"          "None"
+                "add"           "Add"
+                "multiply"      "Multiply"
+                "maximum"       "Maximum"
+                "minimum"       "Minimum"
+                "topounion"     "Activity Union"
+                "topointersect" "Activity Intersection"
+            }
+        }
+        parm {
+            name    "distancemode"
+            label   "Distance Mode"
+            type    ordinal
```

Comment:
op_sdf
SDF VDBs
Levelset VDBs tend to be called SDF VDBs.  Yeah, random TLAs, but it is consistent :>